js/script.js

```diff
@@ -0,0 +1,19 @@
+$(function(){
+	var samsungGalaxyS6 = new Phone("Samsung", 3250, "black");
```

Comment:
try to use loop to create this elements ;) create array with objects, every object should contain single phone properties